src/components/IconButton/__stories__/IconButton.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import IconButton from "../IconButton";
+import DialogContentContainer from "../../DialogContentContainer/DialogContentContainer";
+import { Bolt, Activity, Robot, Doc, Pin, Show, HighlightColorBucket, Broom, Add } from "../../Icon/Icons"
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { BUTTON_GROUP, SPLIT_BUTTON } from "../../../storybook/components/related-components/component-description-map";
+
+import "./IconButton.stories.scss";
+
+<Meta
+  title="Components/IconButton"
+  component={ IconButton }
+/>
+
+<!--- Component template -->
+
+export const iconButtonTemplate = createComponentTemplate(IconButton);
+
+<!--- Component documentation -->
+
+# IconButton
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Please write here the component description
+
+<Canvas>
+  <Story name="Overview" args={{}}>
+    <IconButton ariaLabel="Add Button"/>
```

Comment:
We need to use the iconButtonTemplate for the overview story for creating access to the controls in the sandbox (like in the button story)

Reply:
will change

Reply:
current issue with storybook 6.4

Reply:
what do you mean?